stdlib/3/typing.pyi

```diff
@@ -102,7 +102,7 @@ class Generator(Iterator[_T_co], Generic[_T_co, _T_contra, _V_co]):
     def send(self, value: _T_contra) -> _T_co:...
 
     @abstractmethod
-    def throw(self, typ: BaseException, val: Any=None, tb=None) -> None:...
+    def throw(self, typ: BaseException, val: Any = None, tb: Any = None) -> None:...
```

Comment:
The typ arg can also be an exception instance (then I think no value is allowed and I'm not sure about the tb).